development/using-windows.md

```diff
@@ -27,37 +27,31 @@ Luckily, there is a perfectly working CouchDB installation for Windows:
 Then go to `http://localhost:5984/_utils/#/setup` in Windows and do the single node setup. Once done head back to linux and confirm it works:
 
 ```
-scdf@PC:/mnt/c/Users/stefa/Code/medic$ curl http://localhost:5984/
+yrimal@PC: curl http://localhost:5984/
```

Comment:
Typically you can just use `$` to represent a bash command. I don't think it is necessary to prefix it with a username@machinename.